specification/azureactivedirectory/resource-manager/Microsoft.Aadiam/stable/2017-04-01/azureactivedirectory.json

```diff
@@ -37,6 +37,117 @@
     }
   },
   "paths": {
+    "/providers/microsoft.aadiam/privateLinkForAzureAD": {
+      "get": {
```

Comment:
>get [](start = 7, length = 3)

How many private links can there be? Do you need paging here?

Reply:
Standard ARM limits, 800 per subscription. 

This is the same as what "private link service" resource does. 

https://docs.microsoft.com/en-us/azure/azure-resource-manager/management/azure-subscription-service-limits#private-link-limits

Reply:
You should probably add paging via the x-ms-pageable extension. It's ok to add it even if your service only returns one page right now, but it makes adding paging to the service in the future easier.

---
In reply to: [403390060](https://github.com/Azure/azure-rest-api-specs/pull/8854#discussion_r403390060) [](ancestors = 403390060)

Reply:
makes sense I added x-ms-pageable extension. also @majastrz , @arluca , I had to restructure the changes into separate files to keep it clean.